piano-hero/core/src/main/java/com/novoda/pianohero/Chords.java

```diff
@@ -0,0 +1,12 @@
+package com.novoda.pianohero;
+
+class Chords {
+
+    static final Chord C = new Chord(Notes.C4, Notes.E4, Notes.G4);
+    static final Chord F_2ND_INV = new Chord(Notes.C4, Notes.F4, Notes.A4);
```

Comment:
I dunno the names of these variations off by heart, I checked them [here](http://www.pianochorddictionary.com/chords/g/major-inversion2.shtml)